builder/common/step_ami_region_copy.go

```diff
@@ -202,12 +202,14 @@ func (s *StepAMIRegionCopy) amiRegionCopy(ctx context.Context, state multistep.S
 	if err != nil {
 		return "", snapshotIds, err
 	}
+	t := true
```

Comment:
Might be worth updating step_ami_region_copy_test for this to make sure true is passed in in the units too

Reply:
Good call! I've added this to the tests, now we always check if it is properly set (and from the looks of it, it is!).